bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Carousel.java

```diff
@@ -76,4 +76,14 @@ default boolean getAutopauseDisabled() {
         throw new UnsupportedOperationException();
     }
 
+    /**
+     * Returns an accessible label that describes the carousel.
+     *
+     * @return The value of label for Carousel component
```

Comment:
We should use the same consistent return descriptions for these:
• The value of label for Carousel component
• the value of label for navigation
• The value of label

Would prefer:
• an accessible label for the carousel
• an accessible label for the navigation
• an accessible label for the tabs

Reply:
Done